config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :users, only: [:create, :index, :destroy]
+      resources :users, only: [:create, :index, :destroy] do
```

Comment:
I am not sure that we need there nested projects from users. Project can be as top level resource without user.